### ISSUE TITLE
Provide wallpaper centering support 

### ISSUE THREAD

**Issue by f1reflyyyylmao** [MM2-0/Kvaesitso#397]
As others have already suggested, Muzei is still a popular live wallpaper. I'm using Muzei to rotate my own images, but Kvaesitso always displays the leftmost portion of each image. According to the developers of Muzei, changing this behaviour is a responsibility of the launcher: [Issue](https://github.com/muzei/muzei/issues/62). It would be great if Kvaesitso could provide an option to always display the centre of a live wallpaper.

**Comment by The1Unic** [comment]
Yes please, really had a tough time looking for anyone else facing the same issues as me. I'm using the "Backdrops" wallpaper app and it extremely annoyingly places every wallpaper snapping to the left side of my screen, instead of centering it. If I apply the same wallpaper through my setting's wallpaper feature, it centers perfectly fine. When I use this app with a different launcher it also works fine. I really hope a fix can be implemented for this, as I might have to change launchers otherwise.
